src/ir/function.rs

```diff
@@ -49,6 +49,12 @@ impl FunctionKind {
     }
 }
 
+#[derive(Debug, Clone, Copy)]
+pub enum Linkage {
```

Comment:
Could you document this enum and the variants, please? Travis is complaining because of that.